content/docs/dvclive/ml-frameworks/pytorch-lightning.md

```diff
@@ -61,6 +61,23 @@ checkpointing at all as described in the
 
 - `prefix` - (`None` by default) - string that adds to each metric name.
 
+- `log_model` - (`False` by default) - use
+  [`live.log_artifact()`](/doc/dvclive/live/log_artifact) to log checkpoints
+  created by
+  [`ModelCheckpoint`](https://lightning.ai/docs/pytorch/stable/api/lightning.pytorch.callbacks.ModelCheckpoint.html)
+  and annotate the best checkpoint with `type=model` and `name=best` for use in
+  [Studio model registry]. DVCLive will <abbr>cache</abbr> the checkpoint
+  directory and delete checkpoints from previous <abbr>experiments</abbr>, but
+  you can recover the checkpoints from any experiment using DVC.
+
+  - if `log_model == 'all'`, checkpoints are logged during training.
```

Comment:
_all_ checkpoints are logged during training?

also, what does it practically mean? does it create a record per each checkpoint in `dvc.yaml`?